03 - Variables/assignment_shortcuts.rb

```diff
@@ -0,0 +1,47 @@
+a=5
+a=a+10
+puts a
+a=10
+a+=100
+puts a
+
+b=100
+b=b-50
+b-=20
+puts b 
+
+c=10
+c*=5
+puts c
+
+d=10
+d/=5
+puts d
+
+name="Haris"
+name += " Butt"
+puts name
+
+=begin
```

Comment:
it would be clearer if the comments and code were structured together for example
```
```=begin
Compound Assignment with Addition
Create a variable score and assign it a value of 10. Use a compound assignment to increase the score by 5, and print the result.
=end
score = 10
score += 5
puts "Score value is #{score}"
```